### ISSUE TITLE
Flagception\Bundle\FlagceptionBundle\Profiler\FeatureDataCollector::collect declaration  error in Symfony5

### ISSUE THREAD

**Issue by hanishsingla** [bestit/flagception-bundle#67]
Compile Error: Declaration of Flagception\Bundle\FlagceptionBundle\Profiler\FeatureDataCollector::collect(Symfony\Component\HttpFoundation\Request $request, Symfony\Component\HttpFoundation\Response $response, ?Exception $exception = NULL) must be compatible with Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface::collect(Symfony\Component\HttpFoundation\Request $request, Symfony\Component\HttpFoundation\Response $response, ?Throwable $exception = NULL)

**Comment by migo315** [comment]
Thanks for your report!

I am currently considering how to fix this for all supported Symfony versions. The interface has changed several times:
- https://github.com/symfony/symfony/blob/3.1/src/Symfony/Component/HttpKernel/DataCollector/DataCollectorInterface.php
- https://github.com/symfony/symfony/blob/4.3/src/Symfony/Component/HttpKernel/DataCollector/DataCollectorInterface.php
- https://github.com/symfony/symfony/blob/5.0/src/Symfony/Component/HttpKernel/DataCollector/DataCollectorInterface.php

**Comment by migo315** [comment]
Currently I cannot invest any free time for this issue. Maybe you could provide a solution / pr?

**Comment by hanishsingla** [comment]
Sure, I am working on it. Will send PR soon.

**Comment by migo315** [comment]
The master Branch has full Symfony 5 support now. I've done all changes and skip support for Symfony 2+3+4. I've released this changes with bundle Version 4.0.0.

But I also created the branch "3.x", where we can maintain Symfony 2 + 3 + 4.

So, for Symfony 5 use Flagception Bundle 4.0. For Symfony prior 5 use Flagception Bundle 3.0

Let me know if you find any mistakes. My tests were all successful. :-)
